dialogueMaker/dialogueMaker.cpp

```diff
@@ -12,17 +11,10 @@ void Scene::printScene() {
   /*for (int i = 0; i < sceneLength; i++) {
     std::cout << "-";
   }*/
-  std::cout << "---------------------------------------------------------------------------------------------------------------------------";
-  //std::cout << '\n' << dialogue << '\n';
-  std::cout << '\n';
-  for (int i = 0; i < dialogue.size(); i++)
-  {
-    std::cout << dialogue[i];
-    Sleep(1);
-  }
-  std::cout << '\n';
-  std::cout << "---------------------------------------------------------------------------------------------------------------------------";
-  std::cout << '\n';
+  std::cout << "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------";
+  std::cout << '\n' << dialogue << '\n';
+  std::cout << "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------";
+  std::cout << '\n';\
```

Comment:
Please remove a leftover backslash
```suggestion
  std::cout << '\n';
```